compiler-cli/src/main.rs

```diff
@@ -271,6 +273,15 @@ pub enum ExportTarget {
     },
 }
 
+#[derive(Subcommand, Debug)]
+enum PublishRevertCommand {
+    /// Reverts a published release within 24 hours of initial publish or within one hour for an already existing package
```

Comment:
This is too long to be a CLI help message, it needs to fit in around 60 columns

Reply:
Hmm... then I guess I will have to trim out the information about when a package can be reverted. If instead added a link to the hex docs which talk about the time limit then the link itself would also be too long.

Oh well, I guess the message will just have to say: "Reverts a published release"